import "session_commands.proto";

message Command_ReplayList {
    extend SessionCommand {
        optional Command_ReplayList ext = 1100;
    }
}
